rs/cli/src/commands/registry.rs

```diff
@@ -317,6 +338,10 @@ async fn get_nodes(
     Ok(nodes)
 }
 
+fn get_unassigned_nodes(local_registry: &Arc<dyn LazyRegistry>) -> anyhow::Result<Option<UnassignedNodesConfigRecord>> {
+    local_registry.get_unassigned_nodes()
+}
+
```

Comment:
At this moment, we should remove it, its an empty function. I forgot to remove it in my refactoring.

Reply:
done